spec/factories/activity.rb

```diff
@@ -15,6 +15,7 @@
     geography { :recipient_region }
     recipient_region { "489" }
     recipient_country { nil }
+    intended_beneficiaries { ["CU", "DM", "DO"] }
```

Comment:
How about using the simplest value possible here - maybe even the "not applicable" state, if that is valid?

Reply:
I selected a random array of countries to have more than one country on the factory. I'm happy to reduce the array but "Not Applicable" cannot be used here because `geography` is set to `recipient_region`, hence at least a benefitting country is needed. Do you want me to reduce the array to just one random country?

Reply:
Actually, on second thought, this is stored as an array, so I'd leave it be :+1: